Copy to SD Card root directory to update/config.ini

```diff
@@ -113,6 +113,14 @@ emulated_m600:1
 #   Options: [disable: 0, enable: 1]
 emulated_m109_m190:1
 
+#### G-code File Comment Parsing
+# The TFT parses and processes extra information provided by the slicer as comments in the G-code file.
+#
+# NOTE: Enable it in case the slicer (e.g. Cura) supports extra information.
+#
+#   Options: [disable: 0, enable: 1]
+file_comment_parsing:1
+
```

Comment:
You could mention that for the moment print time and print layer information are parsed and processed from the G-code file (and nothing else). 
Also it is not obvious for a regular user that if file comment parsing is not enabled than "layer display type" option becomes redundant.

Reply:
Just to avoid to provide wrong information/explanation, could you please provide those info or add them in a further PR?